### PR TITLE
build(pnpm-lock): bump zenroom and slangroom versions

### PR DESCRIPTION
**Pull request by matteo-cristino**

### CODE REVIEW

File: .releaserc [hunk 1]
```diff
@@ -6,9 +6,17 @@
...
   "plugins": [
-    "@semantic-release/commit-analyzer",
     "@semantic-release/release-notes-generator",
     "@semantic-release/changelog",
+    [
+      "@semantic-release/commit-analyzer",
+      {
+        "preset": "angular",
+        "releaseRules": [
+          { "type": "build", "scope": "pnpm-lock", "release": "patch" }
```
Comment (puria) [inline .releaserc:16]: I dunno... It's something like:
`build(pnpm-lock): zenroom update`
correct?

**Review by puria — APPROVED** [review]

File: .releaserc (hunk 1 above)
Comment (puria) [reply .releaserc:16]: MAsssssI VA BENE

Comment (matteo-cristino) [reply .releaserc:16]: yes it is like that

Comment (matteo-cristino) [reply .releaserc:16]: maybe more appropriate chore(pnpm-lock)?

**Pull request merged**